features/all-positions/PositionActionsDialog.tsx

```diff
@@ -523,6 +595,13 @@ const PositionActionsDialog = (props: DialogProps) => {
                                 inputProps={{ min: "0" }}
                                 label={`Max collateral/token`}
                                 value={maxCollPerToken}
+                                error={collRatio > Number(maxCollPerToken)}
+                                helperText={
+                                  collRatio > Number(maxCollPerToken) &&
+                                  `Current collateral ratio of ${collRatio.toFixed(
+                                    4
+                                  )} is above the specified maximum ratio, transaction will revert.`
+                                }
                                 onChange={(
                                   e: React.ChangeEvent<HTMLInputElement>
                                 ) => setMaxCollPerToken(e.target.value)}
```

Comment:
do you want to perhaps auto-fill the max tokens to liquidate as the full position size?

Reply:
What if user doesn't have enough balance?

Reply:
I auto filled to max
![Screen Shot 2020-08-27 at 08 17 20](https://user-images.githubusercontent.com/9457025/91441168-bc6b0200-e83d-11ea-9962-a4fed8fc2718.png)

Reply:
We could do the min of the position size and their balance?

Reply:
Yeah @mrice32 that's easiest implementation I think